### PR TITLE
more good status effects via hookah

### PR DESCRIPTION
**Pull request by Veracity0**
I have five characters that equip a hookah when running their turns.

```
All Rainbow-y is good quality, but KoLmafia thought it was neutral
Aloofah is good quality, but KoLmafia thought it was neutral
Anti-Burning is good quality, but KoLmafia thought it was neutral
Anti-Creeped is good quality, but KoLmafia thought it was neutral
Anti-Freezing is good quality, but KoLmafia thought it was neutral
Anti-Goosebumped is good quality, but KoLmafia thought it was neutral
Anti-Hurt is good quality, but KoLmafia thought it was neutral
Anti-Odored is good quality, but KoLmafia thought it was neutral
Ashamed is good quality, but KoLmafia thought it was neutral
Bet Your Autumn Dollar is good quality, but KoLmafia thought it was neutral
Bulging Jaw Muscles is good quality, but KoLmafia thought it was neutral
Burningly Oiled is good quality, but KoLmafia thought it was neutral
Carbonated is good quality, but KoLmafia thought it was neutral
Cosmic Ball in the Air is good quality, but KoLmafia thought it was neutral
Craving Prawns is good quality, but KoLmafia thought it was neutral
Delayed Gratification is good quality, but KoLmafia thought it was neutral
Delighted by Autumn is good quality, but KoLmafia thought it was neutral
Double Carbonated is good quality, but KoLmafia thought it was neutral
Double Frozen is good quality, but KoLmafia thought it was neutral
Double Hot Soupy Garbage is good quality, but KoLmafia thought it was neutral
Doubly Burningly Oiled is good quality, but KoLmafia thought it was neutral
Doubly Craving Prawns is good quality, but KoLmafia thought it was neutral
Doubly Shivering Spine is good quality, but KoLmafia thought it was neutral
Doubly Spookily Greasy is good quality, but KoLmafia thought it was neutral
Earning Interest is good quality, but KoLmafia thought it was neutral
Energized is good quality, but KoLmafia thought it was neutral
Extra-Green is good quality, but KoLmafia thought it was neutral
Feeling Nervous is good quality, but KoLmafia thought it was neutral
Fire-Roasted Feelings is good quality, but KoLmafia thought it was neutral
Five Sticky Fingers is good quality, but KoLmafia thought it was neutral
Flapper Dancin' is good quality, but KoLmafia thought it was neutral
Frozen is good quality, but KoLmafia thought it was neutral
Gaffe Free is good quality, but KoLmafia thought it was neutral
Glowing Hands is good quality, but KoLmafia thought it was neutral
Gritty is good quality, but KoLmafia thought it was neutral
Hot Soupy Garbage is good quality, but KoLmafia thought it was neutral
Loyal as a Rock is good quality, but KoLmafia thought it was neutral
On the Minty Precipice is good quality, but KoLmafia thought it was neutral
On the Precipice of Carrots is good quality, but KoLmafia thought it was neutral
One Foot Heavier is good quality, but KoLmafia thought it was neutral
Pyrite Pride is good quality, but KoLmafia thought it was neutral
Shivering Spine is good quality, but KoLmafia thought it was neutral
Slippery and Speedy is good quality, but KoLmafia thought it was neutral
Spookily Greasy is good quality, but KoLmafia thought it was neutral
Stained is good quality, but KoLmafia thought it was neutral
Stuffed is good quality, but KoLmafia thought it was neutral
Teacher's Pet is good quality, but KoLmafia thought it was neutral
Temporarily Filtered is good quality, but KoLmafia thought it was neutral
Tingling Insides is good quality, but KoLmafia thought it was neutral
Tingly Tongue is good quality, but KoLmafia thought it was neutral
Too Shamed is good quality, but KoLmafia thought it was neutral
Velougated is good quality, but KoLmafia thought it was neutral
Whet Appetite is good quality, but KoLmafia thought it was neutral
Wisdom of Others is good quality, but KoLmafia thought it was neutral
Wisdom of the Autumn Years is good quality, but KoLmafia thought it was neutral
You Have Ever Seen The Rain is good quality, but KoLmafia thought it was neutral
```

### CODE REVIEW

File: src/data/statuseffects.txt [hunk 1]
```diff
@@ -2267,8 +2267,8 @@
 2265
 2266	Nothing Happened	nontini.gif	4ea39859915bf5e160d3e3010d18f37b	neutral	none	drink 1 nothingtini
 2267	Hello, I Must Be Going	petejacket.gif	de2fd12e4cc01c737679abb75caf96dc	neutral	none	drink 1 Phil Collins
-2268	Aloofah	loof_candy.gif	4e1b19f13f36d4e3d03aa14f51e55414	neutral	none	use 1 chocolate-covered loofah
-2269	One Foot Heavier	fs_candy.gif	5a26995fdbca9e91a81e788a76f7b4ba	neutral	none	use 1 Flag by the Foot&trade;
+2268	Aloofah	loof_candy.gif	4e1b19f13f36d4e3d03aa14f51e55414	good	none	use 1 chocolate-covered loofah
+2269	One Foot Heavier	fs_candy.gif	5a26995fdbca9e91a81e788a76f7b4ba	god	none	use 1 Flag by the Foot&trade;
```
Comment (midgleyc) [inline src/data/statuseffects.txt:2271]:
```suggestion
2269	One Foot Heavier	fs_candy.gif	5a26995fdbca9e91a81e788a76f7b4ba	good	none	use 1 Flag by the Foot&trade;
```

File: src/data/statuseffects.txt [hunk 2]
```diff
@@ -2696,35 +2696,35 @@
...
-2703	Glowing Hands	maydayglowstick.gif	70f654ece069988c295d6c1e79737102	neutral	none	use 1 emergency glowstick
-2704	Tingly Tongue	maydaybattery.gif	3543e77d43816e573a802c5c68226f3d	neutral	none	use 1 spare battery
-2705	Temporarily Filtered	maydaymask.gif	3c274a686e3d4364c0ae2aed6f80c877	neutral	none	use 1 single-use dust mask
-2706	Gaffe Free	maydaytape.gif	af3ed07a138fe185660609606c1c2926	neutral	none	use 1 gaffer's tape
-2707	On the Minty Precipice	maydaybar.gif	6a543864b4bb1b558217729a507a4044	neutral	none	eat 1 cool mint precipice bar
-2708	On the Precipice of Carrots	maydaybar.gif	9b93c7a0436a4594d28bd8d03952cd13	neutral	none	eat 1 carrot cake precipice bar
+2703	Glowing Hands	maydayglowstick.gif	70f654ece069988c295d6c1e79737102	good	none	use 1 emergency glowstick
+2704	Tingly Tongue	maydaybattery.gif	3543e77d43816e573a802c5c68226f3d	good	none	use 1 spare battery
+2705	Temporarily Filtered	maydaymask.gif	3c274a686e3d4364c0ae2aed6f80c877	good	none	use 1 single-use dust mask
+2706	Gaffe Free	maydaytape.gif	af3ed07a138fe185660609606c1c2926	frozen	none	use 1 gaffer's tape
```
Comment (midgleyc) [inline src/data/statuseffects.txt:2706]:
```suggestion
2706	Gaffe Free	maydaytape.gif	af3ed07a138fe185660609606c1c2926	good	none	use 1 gaffer's tape
```

**Review by midgleyc — CHANGES_REQUESTED** [review]
Guess we could do with either a test or the parser to confirm these values are one of the allowed ones.

**Comment by Veracity0** [comment]
> Guess we could do with either a test or the parser to confirm these values are one of the allowed ones.

I could have sworn we had a DataFilesIntegrityTest which does exactly that.

File: src/data/statuseffects.txt (hunk 1 above)
Comment (Veracity0) [reply src/data/statuseffects.txt]: I made DataFileTest detect erors.
Fixed.

File: src/data/statuseffects.txt (hunk 2 above)
Comment (Veracity0) [reply src/data/statuseffects.txt]: I made DataFileTest detect erors.
Fixed.

**Review by midgleyc — APPROVED** [review]
Thanks!

**Pull request merged**